src/odemis/driver/simcam.py

```diff
@@ -118,7 +118,7 @@ def clip_max_res(img_res):
                                               cls=(int, long), setter=self._setTranslation)
 
         self._orig_exp = self._img.metadata.get(model.MD_EXP_TIME, 0.1)  # s
-        self.exposureTime = model.FloatContinuous(self._orig_exp, (1e-3, 10), unit="s")
+        self.exposureTime = model.FloatContinuous(self._orig_exp, (1e-3, 20), unit="s")
```

Comment:
Instead of adjusting the code to fit a specific data, I'd suggest to make the code automatically fit, based on the current exposure time of the image. Something like this:
`self.exposureTime = model.FloatContinuous(self._orig_exp, range=(1e-3, max(10, self._orig_exp * 2)), unit="s")`

Reply:
that's a good one, i made the change